### PR TITLE
Remove unused script from documentation generation

### PR DESCRIPTION
**Pull request by s-zeng**
scripts/convert.py is no longer in use - references to it should be removed from the README.

On a side note, is there any source of truth about how to contribute to this repository, in terms of the actual content of the dhall files? The one reference I found in the docs here is to convert.py, which is no longer in the repo. I wanted to take a shot at fixing [this](https://github.com/dhall-lang/dhall-kubernetes/issues/136#issuecomment-715646180) but I found nothing that could point me to actually accomplishing that. Thanks in advance for any help.

### CODE REVIEW

**Comment by Gabriella439** [comment]
@s-zeng: I created a pull request to update the `README` to answer your contribution-related questions: https://github.com/dhall-lang/dhall-kubernetes/pull/154

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ In the following example, we:
 -- examples/deploymentSimple.dhall
 
 let kubernetes =
-      https://raw.githubusercontent.com/dhall-lang/dhall-kubernetes/master/package.dhall sha256:d541487f153cee9890ebe4145bae8899e91cd81e2f4a5b65b06dfc325fb1ae7e
+      ../package.dhall sha256:d541487f153cee9890ebe4145bae8899e91cd81e2f4a5b65b06dfc325fb1ae7e
```
Comment (Gabriella439) [inline README.md]: We probably want to leave these example imports as remote imports.  See #140 and #142

**Comment by s-zeng** [comment]
Fixed. Thanks for the headsup

**Comment by Gabriella439** [comment]
@s-zeng: Thank you for fixing that! 🙂

**Pull request merged**